spec/support/features/page_object/new_messageboard.rb

```diff
@@ -30,7 +30,7 @@ def showing_the_form?
     end
 
     def on_the_messageboard_list?
-      has_css? 'body#thredded_messageboards_index'
+      has_css? 'body#thredded-messageboards-index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.